tests/main/views/test_brief_response.py

```diff
@@ -688,16 +688,19 @@ def test_list_empty_brief_responses(self):
         assert data['briefResponses'] == []
         assert 'self' in data['links'], data
 
-    def test_list_brief_responses(self):
+    @pytest.mark.parametrize("with_data_false", (False, True))
```

Comment:
This var name is hurting my brain a bit with the prospect of `with_data_false` equalling `True` - how about `data_not_included`?